service/discovery/azure/compute.go

```diff
@@ -279,6 +277,64 @@ func (d *azureComputeDiscovery) handleFunction(function *armappservice.Site) voc
 	}
 }
 
+func (d *azureComputeDiscovery) handleWebApp(webApp *armappservice.Site) voc.IsCompute {
+	var (
+		tlsVersion = ""
+		enc        *voc.TransportEncryption
+	)
+
+	// If a mandatory field is empty, the whole function is empty
+	if webApp == nil {
+		return nil
+	}
+
+	switch util.Deref(webApp.Properties.SiteConfig.MinTLSVersion) {
+	case armappservice.SupportedTLSVersionsOne2:
+		tlsVersion = constants.TLS1_2
+	case armappservice.SupportedTLSVersionsOne1:
+		tlsVersion = constants.TLS1_1
+	case armappservice.SupportedTLSVersionsOne0:
+		tlsVersion = constants.TLS1_0
+	}
+
+	if tlsVersion != "" {
+		enc = &voc.TransportEncryption{
+			Enforced:   util.Deref(webApp.Properties.HTTPSOnly),
+			TlsVersion: tlsVersion,
+			Algorithm:  AES256,
```

Comment:
Doesn't have to be AES256, see also:
See https://techcommunity.microsoft.com/t5/apps-on-azure-blog/min-tls-cipher-suite-preview-now-available-on-azure-portal-and/ba-p/3804134

I think what we actually want here is a list of possible cipher suites the resource with transport encryption (here web app) allows?

Reply:
It should be there now with release 2.3.0. See https://github.com/Azure/azure-sdk-for-go/blob/0d3efbd9bb2603eed4e91e71379d11ad43ead15a/sdk/resourcemanager/appservice/armappservice/CHANGELOG.md?plain=1#L18

I think we are using a very old version of this module (1.x vs. 2.x)

Reply:
I added the cipher suits to WebApps and functions.